config/index.js

```diff
@@ -1,7 +1,12 @@
 const { validateSchema } = require('./schema')
 require('dotenv').config()
 
```

Comment:
Whilst working on this - I noticed that my toBool function would convert an undefined (non existent) env variable to false, which is not what we want - so I made the function also implicitly check for true OR false.